karapace/karapace.py

```diff
@@ -79,7 +80,10 @@ async def root_get(self) -> NoReturn:
         self.r({}, "application/json")
 
     async def health(self, _request: Request) -> aiohttp.web.Response:
-        resp: JsonObject = {"process_uptime_sec": int(time.monotonic() - self._process_start_time)}
+        resp: JsonObject = {
+            "process_uptime_sec": int(time.monotonic() - self._process_start_time),
+            "Karapace-Version": __version__,
```

Comment:
```suggestion
            "karapace_version": __version__,
```

issue: Let's follow same casing as the other healthcheck keys.